pcsx2/gui/AppMain.cpp

```diff
@@ -563,6 +586,27 @@ void Pcsx2App::LogicalVsync()
 		}
 	}
 
+	if (g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
+		if (EnableFMV == 1) {
+#ifdef PCSX2_DEVBUILD 
+			Console.Warning("FMV on");
+#endif
+			Do43Switch(1);
```

Comment:
Would be nice to use ``true`` for ``1`` and ``false`` for ``0`` on function call for ``Do43Switch ()``. It's a Bool parameter after all ;)